spec/rubocop/cop/lint/useless_method_definition_spec.rb

```diff
@@ -170,6 +214,14 @@ def method(x: 1)
     RUBY
   end
 
+  it 'does not register an offense when method definition with non access modifier containing only `super` call' do
```

Comment:
Technically speaking if it's not an access modifier we should probably call it should else - e.g. "generic method macro" or whatever. I'm not sure what's the best terminology here.

Reply:
I'm not sure the best terminology, but I think "generic method macro" is definitely a better name. Thank you for the suggestion! I've updated this PR.